scripts/psnr-test.sh

```diff
@@ -16,11 +16,20 @@
 #
 
 SCRIPT_DIR=$(dirname $0)
-OUTPUT_FRAME=$1
-INPUT_FRAME_DIR=$2
-RESIZED_FRAME_DIR=$3
+OUTPUT_FRAME="$1"
+INPUT_FRAME_DIR="$2"
+RESIZED_FRAME_DIR="$3"
+QR_CODE_DECODER="zbarimg"
 
-FRAME_NUMBER=$(java -jar $SCRIPT_DIR/javase-3.2.2-SNAPSHOT-jar-with-dependencies.jar $OUTPUT_FRAME |head -3|tail -1)
```

Comment:
Do we still need that jar if we're using zbarimg instead?

Reply:
no...and i think you're right, we should remove it.  i'll delete it.